README.md

```diff
@@ -23,14 +23,16 @@ saucectl configure
 
 ## Running The Examples
 
-Simply check out this repo and run the appropriate command below :rocket:
+Simply check out this repo, set your XCUITest test bundles and run the appropriate command below :rocket:
 
```

Comment:
Just putting the statement a bit closer to where you most likely will need it. The moment the user reads "set your XCUITest test bundles", they'll immediately ponder how to get to those, if all they've been using is the Xcode IDE without every inspecting where the build files end up. For those too impatient to read all of the instructions :D 

```suggestion
:bulb: We also provide [DemoApp](DemoApp/) to demonstrate the test workflow. Click [here](.github/workflows/test.yml) for more details on how to build your XCUITest test bundle.
```